www/index.ejs

```diff
@@ -3,30 +3,16 @@
 <head>
   <meta charset="utf-8">
   <meta http-equiv="x-ua-compatible" content="ie=edge">
-  <title><%= config.title %></title>
   <meta name="description" content="">
   <meta name="viewport" content="width=device-width, initial-scale=1">
-  <link rel="apple-touch-icon" href="/apple-touch-icon.png">
-
-  <link rel="icon" type="image/png" href="/favicon-96x96.png" sizes="96x96">
-  <link rel="icon" type="image/png" href="/favicon-16x16.png" sizes="16x16">
-
-  <link rel="manifest" href="/manifest.json">
 
   <meta name="apple-mobile-web-app-capable" content="yes">
-  <meta name="application-name" content="Integreat">
-  <meta name="apple-mobile-web-app-title" content="Integreat">
-  <meta name="apple-itunes-app" content="app-id=1072353915">
   <meta name="apple-mobile-web-app-status-bar-style" content="black-translucent">
 
   <meta name="mobile-web-app-capable" content="yes">
-  <meta name="theme-color" content="#fbda16">
-  <meta name="msapplication-navbutton-color" content="#fbda16">
   <meta name="msapplication-starturl" content="/">
   <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
 
-  <link rel="icon" type="image/png" sizes="96x96" href="/favicon-96x96.png">
-  <link rel="apple-touch-icon" type="image/png" sizes="96x96" href="/favicon-96x96.png">
   <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
```

Comment:
Do we use this in the neuburg front-end or the integreat-webapp?

Reply:
In neuburg-frontend only